src/RoundManager.sol

```diff
@@ -424,7 +424,7 @@ contract RoundManager is
         if (escrow.status != EscrowStatus.PAID) revert DealNotPaid();
         if (escrow.corpAssets.length > 0) revert AlreadyAllocated();
 
-        (uint256 tokenId, uint256[] memory certIds, uint256 refund) = RoundManagerStorage.allocate(
+        (uint256 tokenId, uint256[] memory certIds, uint256 usedAmount, uint256 refund) = RoundManagerStorage.allocate(
```

Comment:
This way we get both `usedAmount` and `refund` at the same time, seems more coherent; and we could avoid one storage access below